staging/src/k8s.io/component-base/logs/logs.go

```diff
@@ -33,12 +33,21 @@ var logFlushFreq = pflag.Duration(logFlushFreqFlagName, 5*time.Second, "Maximum
 
 func init() {
 	klog.InitFlags(flag.CommandLine)
+	initLogRegistry()
+
+	options := NewOptions()
+	options.AddFlags(pflag.CommandLine)
+	if err := options.Validate(); err != nil {
+		return
+	}
+	options.Apply()
```

Comment:
```suggestion
```
We should not force this in whole codebase. Components themselfs will be responsible for adding logs.Options to their Options struct and initializing it properly